editoast/schemas/src/infra/level_crossing.rs

```diff
@@ -0,0 +1,52 @@
+use educe::Educe;
+use serde::Deserialize;
+use serde::Serialize;
+use utoipa::ToSchema;
+
+use crate::primitives::Identifier;
+use crate::primitives::OSRDIdentified;
+
+#[derive(Debug, Default, Clone, Deserialize, Serialize, PartialEq, ToSchema)]
+#[serde(deny_unknown_fields)]
+pub struct LevelCrossing {
+    #[schema(inline)]
+    pub id: Identifier,
+    pub name: String,
+    pub short_zone_length: f64,
+    pub parts: Vec<LevelCrossingPart>,
+    #[serde(default)]
+    #[schema(inline)]
+    pub extensions: LevelCrossingExtension,
```

Comment:
Do we want to continue introducing extensions for infra objects? We're starting to see the limits of this model with OPs, and might want to reverse course there.

Is the kilometric point really that specific to SNCF and not useful elsewhere?